playbooks/byo/etcd/filter_plugins

```diff
@@ -0,0 +1 @@
+../../../filter_plugins/
```

Comment:
Is this a symlink like the other playbooks, or an actual file?  Same question on the other roles and plugins below